cc.go

```diff
@@ -0,0 +1,175 @@
+package sccp
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/wmnsk/go-sccp/params"
+)
+
+/*
+Message type 2.1 F 1
+Destination local reference 3.2 F 3
+Source local reference 3.3 F 3
+Protocol class 3.6 F 1
+Credit 3.10 O 3
+Called party address 3.4 O 4 minimum
+Data 3.16 O 3-130
+Importance 3.19 O 3
+End of optional parameter 3.1 O 1
+*/
+type CC struct {
+	Type                      MsgType
+	DestinationLocalReference params.LocalReference
+	SourceLocalReference      params.LocalReference
+	params.ProtocolClass
+
+	Opts []*params.Optional
+
+	Data               *params.Optional
+	CalledPartyAddress *params.PartyAddress
```

Comment:
Handling of the optional parameters could be nicer, but it's due to the bad-designed parameter handling in the current code base. I will work on refactoring parameters.

Reply:
Sorry, let me comment on this

In fact, for the practical case, I just need convenient access to Data and Called/Calling pty-s only.
To my knowledge, the only interface where connection-oriented SCCP is used, is GSM A-interface, and I can hardly imagine the real use of other params there.
I would appreciate to avoid making things complicated

Reply:
Q.713 is such a small spec and I'm more comfortable to just have everything implemented rather than to ignore some of them. I think I can quickly make it done soon-ish.